keras_cv/models/models_test.py

```diff
@@ -283,6 +283,42 @@ def test_application_variable_input_channels(self, app, last_dim):
 
         backend.clear_session()
 
+    @parameterized.parameters(*MODEL_LIST)
+    def test_rescaling(self, app, _):
+        for include_rescaling in [True, False]:
+            if "Mixer" not in app.__name__:
+                model = app(
+                    input_shape=(224, 224, 3),
+                    include_top=True,
+                    num_classes=1000,
+                    include_rescaling=include_rescaling,
+                    weights=None,
+                )
+            elif "MixerB16" in app.__name__ or "MixerL16" in app.__name__:
+                model = app(
+                    input_shape=(224, 224, 3),
+                    patch_size=(16, 16),
+                    include_top=True,
+                    num_classes=1000,
+                    include_rescaling=include_rescaling,
+                    weights=None,
+                )
+            elif "MixerB32" in app.__name__:
+                model = app(
+                    input_shape=(224, 224, 3),
+                    patch_size=(32, 32),
+                    include_top=True,
+                    num_classes=1000,
+                    include_rescaling=include_rescaling,
+                    weights=None,
+                )
+
+            if include_rescaling:
```

Comment:
lets have one test with & without:

```
test_rescaling_true
```

```
test_rescaling_false
```

to ensure we hit both cases for all models.

Reply:
I realized these cases are covered by existing tests, so I moved the assertions into those cases.